app/src/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2554,6 +2592,18 @@ private int getFontSizeInPx() {
                 getResources().getDisplayMetrics());
     }
 
+    private void reportFormEntry() {
+        TimedStatsTracker.registerEnterForm(getCurrentFormTitle());
+    }
+
+    private void reportFormExit() {
+        TimedStatsTracker.registerExitForm(getCurrentFormTitle());
+    }
```

Comment:
I was thinking of this as just the average time spent in a single form entry session, regardless of whether or not the form was completed/saved. But the right answer really just depends on how Kevin/Amelia were intending to use this metric, so will defer to them. @dimagi/product -- Would you guys rather this include all form entry sessions, or only those that resulted in form save or completion?